<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CpuPlot" name="plotCpu">
    <property name="geometry">
     <rect>
      <x>979</x>
      <y>100</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="MemBwPlot" name="plotMemory">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>100</y>
      <width>450</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelCpu">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>94</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU Utilization:</string>
    </property>
   </widget>
   <widget class="Line" name="lineHorizontal1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1920</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineHorizontal0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border-width: 1px;
     border-style: solid;
     border-radius: 4px;
 }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineVertical0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>20</width>
      <height>68</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxGuiTransparency">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>GUI Transparency</string>
    </property>
    <widget class="QSlider" name="sliderGuiTransparency">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelGuiTransparency">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>31</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterMode">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Mode</string>
    </property>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>167</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutFilterMode">
      <item>
       <widget class="QRadioButton" name="radioButtonFilterModeOff">
        <property name="text">
         <string> OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterModeSw">
        <property name="text">
         <string> SW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFilterModeHw">
        <property name="text">
         <string> HW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLogoDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>450</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataFlowDisplay">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>100</y>
      <width>450</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxPerformance">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>10</y>
      <width>287</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Performance</string>
    </property>
    <widget class="QWidget" name="layoutWidget3">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>21</y>
       <width>255</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutPerformance" columnminimumwidth="0,50,30,0,50,0">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPerformanceCpu0">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>CPU0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelPerformanceCpu0Val">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelPerformanceCpu0Units">
        <property name="text">
         <string>[%]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelPerformanceHp0">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>HP0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelPerformanceHp0Val">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelPerformanceHp0Units">
        <property name="text">
         <string>[Gb/s]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPerformanceCpu1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>CPU1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelPerformanceCpu1Val">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelPerformanceCpu1Units">
        <property name="text">
         <string>[%]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelPerformanceHp2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>HP2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="labelPerformanceHp2Val">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelPerformanceHp2Units">
        <property name="text">
         <string>[Gb/s]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelMemory">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>80</y>
      <width>169</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Throuput HP Ports:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSobelControls">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Sobel Controls </string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>21</y>
       <width>102</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutSobelControlsThreshold">
      <item>
       <widget class="QLabel" name="labelSobelControlsEdgeSensitivity">
        <property name="text">
         <string>Edge Sensitivity</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutControlsThreshold">
        <item>
         <widget class="QSlider" name="sliderSobelControlsThreshold">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSobelControlsThreshold">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBoxSobelControlsInvert">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>  Invert  </string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="lineVertical1">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>10</y>
      <width>20</width>
      <height>68</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxVideoSource">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Source</string>
    </property>
    <widget class="QComboBox" name="comboBoxVideoSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFilterType">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Type</string>
    </property>
    <widget class="QComboBox" name="comboBoxFilterType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxVideoControls">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Controls</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>166</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonVideoControlsOff">
        <property name="text">
         <string> OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonVideoControlsOn">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonVideoControlsAuto">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTestPattern">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Pattern</string>
    </property>
    <widget class="QComboBox" name="comboBoxTestPattern">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>153</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBaseTrdName">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Zynq Base TRD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBaseTrdReleaseVersion">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>20</y>
      <width>44</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMinMax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CpuPlot</class>
   <extends>QFrame</extends>
   <header>cpuplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemBwPlot</class>
   <extends>QFrame</extends>
   <header>membwplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
